tests/unit/test_protocol.py

```diff
@@ -130,6 +130,27 @@ def test_column_date():
     assert protocol.build_date(date(2019, 1, 1)) == b'\x00\x00\x00\x04\x00\x00\x1b\x1c'
 
 
+def test_timestamp():
```

Comment:
IMO we should test timestamp and timestamp_tz separately

Reply:
done